README.md

```diff
@@ -167,37 +167,37 @@ $ brew install git-xargs
 1. **Ensure you have Golang installed and working properly on your system.** [Follow the official Golang install guide](https://golang.org/doc/install) to get started.
 
 1. **Run go get to install the latest release of git-xargs**:
-     ```bash
-     go get github.com/gruntwork-io/git-xargs
-     ```
+   ```bash
+   go get github.com/gruntwork-io/git-xargs
+   ```
 
 1. **Alternatively, use go get to install a specific release of git-xargs**:
 
-     ```bash
-     go get github.com/gruntwork-io/git-xargs@v0.0.5
-     ```
+   ```bash
+   go get github.com/gruntwork-io/git-xargs@v0.0.5
+   ```
 
 ### Try it out!
 
 1. **Export a valid GitHub token**. See the guide on [Github personal access
    tokens](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token)
    for information on how to generate one. For example, on Linux or Mac, you'd run:
 
-      ```bash
-      export GITHUB_OAUTH_TOKEN=<your-secret-github-oauth-token>
-      ```
+   ```bash
+   export GITHUB_OAUTH_TOKEN=<your-secret-github-oauth-token>
+   ```
 
 1. **Provide a script or command and target some repos**. Here's a simple example of running the `touch` command in
    every repo in your GitHub organization. Follow the same pattern to start running your own scripts and commands
    against your own repos!
 
-      ```bash
-      git-xargs \
-        --branch-name "test-branch" \
-        --commit-message "Testing git-xargs" \
-        --github-org <enter-your-github-org-name> \
-        touch git-xargs-is-awesome.txt
-      ```
+   ```bash
+   git-xargs \
+     --branch-name "test-branch" \
+     --commit-message "Testing git-xargs" \
+     --github-org <enter-your-github-org-name> \
+     touch git-xargs-is-awesome.txt
+   ```
```

Comment:
```suggestion
      ```bash
      git-xargs \
        --branch-name "test-branch" \
        --commit-message "Testing git-xargs" \
        --github-org <enter-your-github-org-name> \
        touch git-xargs-is-awesome.txt
      ```
```